app/apps/new/template.hbs

```diff
@@ -1,39 +1,56 @@
-<div class="row">
-  <div class="col-xs-6">
-    <div class="panel panel-default panel-subdued">
-      <div class="panel-heading">
-        <h3>Create a new app</h3>
-      </div>
-      <div class="panel-body">
-        <form role="form">
-          {{#unless model.isValid}}
-            <div class="alert alert-warning fade in">
-              <a class="close" data-dismiss="alert">×</a>
-              {{#each model.errors.messages as |message|}}
-                <p>{{message}}</p>
-              {{/each}}
-            </div>
-          {{/unless}}
+{{#if session.currentUser.verified}}
```

Comment:
I wonder if we can easily refactor this into `renderTemplate` calls on the route.